constants.py

```diff
@@ -36,3 +37,7 @@
     DATA: {MESSAGE: "Not authorized", IS_SUCCESS: False},
     STATUS: status.HTTP_401_UNAUTHORIZED,
 }
+
+RECORD_NF = "Record not found"
+DELETE_SUCCESS = "Delete successfull"
+DELETE_FAIL = "Delete Fail"
```

Comment:
(case consistent with other messages)
```suggestion
DELETE_FAIL = "Delete failed"
```